src/reasoning/SWRL/swrl.pl

```diff
@@ -200,6 +215,29 @@
 swrl_builtin_pl(endsWith(A,X), atom_concat(_, X_atom, A_atom), Vars) :-
   swrl_atoms([A,X], [A_atom,X_atom], Vars).
 
+%% extensions
+% Note: built-in arguments are passed individually. I have not found a way to 
+% pass them as a list. As a workaround I am adding support for up to 5 arguments.
+swrl_builtin_pl(makeOWLIndividual(A, B),
```

Comment:
Can you add a comment what the arguments are? From the comment it is unclear why built-in arguments are passed at all. Are these al types of the individual?

Reply:
Shouldn't be some kind of rule id be included in the pattern? In case multiple rules would have the same pattern